### DISCUSSION TITLE
Expand output from models with limits to 4k tokens.

### DISCUSSION THREAD

**Discussion by RaySinner** [cline/cline#682] [Feature Requests]
It would be great if you taught your program to ask to continue writing code in those moments when class editing abruptly ends due to the exhaustion of the token limit for the model to respond. For example, the super powerful deepseek model has a limit of 4k tokens per response, and it would be great if you taught how to edit large files with several requests, for example with an automatic command to continue from the same place where you left off

**Comment by anaximanrus** [comment]
Maybe add a feature like Chatgpt's continue generating feature if  @RaySinner's suggestion is harder to develop.
